pkg/agent/openflow/pipeline.go

```diff
@@ -1687,6 +1698,15 @@ func (c *client) conjunctiveMatchFlow(tableID binding.TableIDType, matchKey *typ
 // defaultDropFlow generates the flow to drop packets if the match condition is matched.
 func (c *client) defaultDropFlow(tableID binding.TableIDType, matchKey *types.MatchKey, matchValue interface{}) binding.Flow {
 	fb := c.pipeline[tableID].BuildFlow(priorityNormal)
+	if c.enableDenyTracking {
```

Comment:
This defautDropFlow is for K8s network policy.. is that correct?

Reply:
Yes